Master

SimeonB opened the pull request.

zzeni commented on Simeon/lesson12/fuckbootstrap.html:
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
`fuckbootstrap` ?.. 🐹

zzeni commented on final-test-master/task2/script.js:
```diff
@@ -0,0 +1,59 @@
+function filtererd(text){
+	for (var i = 0; i < single.length; i++)
+		if (text(i) < 5) {
+			return 
```
това е синтактична грешка. Не може просто така да оставиш един `return` statement без стойност, която да връща.
Иначе логиката на цикъла е почти вярна.

zzeni commented on final-test-master/task2/script.js:
```diff
@@ -0,0 +1,59 @@
+var single = text.split(" ");
+function filtererd(text){
+	for (var i = 0; i < single.length; i++)
+		if (text(i) < 5) {
```
това е грешка -  `text` не е функция, а е стринг.
Ти итерираш по списъка `single`. За да провериш дали поредната дума в `single` е по къса от 5 символа, трябва да напишеш: `if (single[i].length < 5)`
(_забележи че се използват квадратни, а не кръгли скоби за достъп до елемента_)

SimeonB commented:
Благодаря за отговорите Жени, за 4 точки не ми достига но следващия път ще
се справя по-добре! До четвъртък :)) !
Поздрави Мони


On Jun 1, 2017 10:32 PM, "Evgenia Manolova" <notifications@github.com>
wrote:

> *@zzeni* commented on this pull request.
>
> *Оценка:*
>
>    -
>
>    Зад.1
>    - (a) - 8т.
>       - (b) - 3т.
>       - (c) - 4т.
>       - (d) - 0т.
>       - (e) - 0т.
>    -
>
>    Зад.2
>    - (1) - 4т.
>       - (2) - 1т.
>       - (3) - 0т.
>       - (4) - 0т.
>       - (5) - 6т.
>    -
>
>    Зад. 3: 18т.
>
> Общо: 46т.
> ------------------------------
>
> In final-test-master/task2/script.js
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#discussion_r119707471>
> :
>
> > +"use strict"
> +
> +// задача 1 (4т.)
> +// Сортирайте следния списък от числа във възходящ ред:
> +var arr = [2, 5, 8, 4, 1, 12];
> +[2, 5, 8, 4, 1, 12].sort(function(a,b) {   return a - b; });
> +//...
> +
> +// задача 2 (4т.)
> +// Напишете функция, която да изважда всички думи, с дължина над 4 символа от следния текст:
> +var text = "The quick brown fox jumps over the lazy dog";
> +var single = text.split(" ");
> +function filtererd(text){
> +	for (var i = 0; i < single.length; i++)
> +		if (text(i) < 5) {
> +			return
>
> това е синтактична грешка. Не може просто така да оставиш един return
> statement без стойност, която да връща.
> Иначе логиката на цикъла е почти вярна.
> ------------------------------
>
> In final-test-master/task2/script.js
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#discussion_r119707975>
> :
>
> > @@ -0,0 +1,59 @@
> +"use strict"
> +
> +// задача 1 (4т.)
> +// Сортирайте следния списък от числа във възходящ ред:
> +var arr = [2, 5, 8, 4, 1, 12];
> +[2, 5, 8, 4, 1, 12].sort(function(a,b) {   return a - b; });
> +//...
> +
> +// задача 2 (4т.)
> +// Напишете функция, която да изважда всички думи, с дължина над 4 символа от следния текст:
> +var text = "The quick brown fox jumps over the lazy dog";
> +var single = text.split(" ");
> +function filtererd(text){
> +	for (var i = 0; i < single.length; i++)
> +		if (text(i) < 5) {
>
> това е грешка - text не е функция, а е стринг.
> Ти итерираш по списъка single. За да провериш дали поредната дума в single
> е по къса от 5 символа, трябва да напишеш: if (single[i].length < 5)
> (*забележи че се използват квадратни, а не кръгли скоби за достъп до
> елемента*)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#pullrequestreview-41613270>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AZBJHeOaK5khwhvYTJfCE3u28eJ6fbMjks5r_xG0gaJpZM4NqtqW>
> .
>

zzeni commented:
Привет. Ще ви дам задачи за повишаване на оценката. Споко, има хора и с 
още по-нисък резултат даже ;)


On 6/1/17 11:22 PM, SimeonB wrote:
> Благодаря за отговорите Жени, за 4 точки не ми достига но следващия 
> път ще
> се справя по-добре! До четвъртък :)) !
> Поздрави Мони
>
>
> On Jun 1, 2017 10:32 PM, "Evgenia Manolova" <notifications@github.com>
> wrote:
>
> > *@zzeni* commented on this pull request.
> >
> > *Оценка:*
> >
> > -
> >
> > Зад.1
> > - (a) - 8т.
> > - (b) - 3т.
> > - (c) - 4т.
> > - (d) - 0т.
> > - (e) - 0т.
> > -
> >
> > Зад.2
> > - (1) - 4т.
> > - (2) - 1т.
> > - (3) - 0т.
> > - (4) - 0т.
> > - (5) - 6т.
> > -
> >
> > Зад. 3: 18т.
> >
> > Общо: 46т.
> > ------------------------------
> >
> > In final-test-master/task2/script.js
> > 
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#discussion_r119707471>
> > :
> >
> > > +"use strict"
> > +
> > +// задача 1 (4т.)
> > +// Сортирайте следния списък от числа във възходящ ред:
> > +var arr = [2, 5, 8, 4, 1, 12];
> > +[2, 5, 8, 4, 1, 12].sort(function(a,b) { return a - b; });
> > +//...
> > +
> > +// задача 2 (4т.)
> > +// Напишете функция, която да изважда всички думи, с дължина над 4 
> символа от следния текст:
> > +var text = "The quick brown fox jumps over the lazy dog";
> > +var single = text.split(" ");
> > +function filtererd(text){
> > + for (var i = 0; i < single.length; i++)
> > + if (text(i) < 5) {
> > + return
> >
> > това е синтактична грешка. Не може просто така да оставиш един return
> > statement без стойност, която да връща.
> > Иначе логиката на цикъла е почти вярна.
> > ------------------------------
> >
> > In final-test-master/task2/script.js
> > 
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#discussion_r119707975>
> > :
> >
> > > @@ -0,0 +1,59 @@
> > +"use strict"
> > +
> > +// задача 1 (4т.)
> > +// Сортирайте следния списък от числа във възходящ ред:
> > +var arr = [2, 5, 8, 4, 1, 12];
> > +[2, 5, 8, 4, 1, 12].sort(function(a,b) { return a - b; });
> > +//...
> > +
> > +// задача 2 (4т.)
> > +// Напишете функция, която да изважда всички думи, с дължина над 4 
> символа от следния текст:
> > +var text = "The quick brown fox jumps over the lazy dog";
> > +var single = text.split(" ");
> > +function filtererd(text){
> > + for (var i = 0; i < single.length; i++)
> > + if (text(i) < 5) {
> >
> > това е грешка - text не е функция, а е стринг.
> > Ти итерираш по списъка single. За да провериш дали поредната дума в 
> single
> > е по къса от 5 символа, трябва да напишеш: if (single[i].length < 5)
> > (*забележи че се използват квадратни, а не кръгли скоби за достъп до
> > елемента*)
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > 
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#pullrequestreview-41613270>,
> > or mute the thread
> > 
> <https://github.com/notifications/unsubscribe-auth/AZBJHeOaK5khwhvYTJfCE3u28eJ6fbMjks5r_xG0gaJpZM4NqtqW>
> > .
> >
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub 
> <https://github.com/zzeni/swift-academy-homeworks/pull/214#issuecomment-305609475>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AAR-wGOxG0C9BoU4Lx41VEuLyw4AegkRks5r_x2WgaJpZM4NqtqW>.
>

-- 
Evgenia Manolova
http://zenlabs.pro
